packages/synapse-react-client/src/lib/containers/download_list_v2/MeetAccessRequirementCard.tsx

```diff
@@ -27,25 +25,17 @@ export const LOCK_TITLE = 'Access Restricted'
 export const MeetAccessRequirementCard: React.FunctionComponent<
   MeetAccessRequirementCardProps
 > = ({ accessRequirementId, count }: MeetAccessRequirementCardProps) => {
-  const handleError = useErrorHandler()
-  const {
-    data: ar,
-    isFetching,
-    isError,
-    error: newError,
-  } = useGetAccessRequirements(accessRequirementId)
-  useEffect(() => {
-    if (isError && newError) {
-      handleError(newError)
-    }
-  }, [isError, newError, handleError])
+  const { data: ar, isLoading } = useGetAccessRequirements(
+    accessRequirementId,
+    { useErrorBoundary: true },
```

Comment:
The `useErrorBoundary: true` option is functionally identical to calling `handleError` with the error object. Since this will throw right away, we don't need to worry about handling cases where `isError` is true.